pkg/scheduler/nodeinfo/node_info.go

```diff
@@ -52,11 +52,14 @@ type NodeInfo struct {
 	podsWithAffinity []*v1.Pod
 	usedPorts        HostPortInfo
 
-	// Total requested resource of all pods on this node.
-	// It includes assumed pods which scheduler sends binding to apiserver but
-	// didn't get it as scheduled yet.
+	// Total requested resources of all pods on this node. This includes assumed
+	// pods, which scheduler has sent for binding, but may not be scheduled yet.
 	requestedResource *Resource
-	nonzeroRequest    *Resource
+	// Total requested resources of all pods on this node with a minimum value
```

Comment:
Would it be better to add an explanation to minimum value: `Total requested resources of all pods on this node with a default value(DefaultMilliCPURequest/DefaultMemoryRequest) applied to each container's CPU and memory requests if they are empty.`